t/resty/options.t

```diff
@@ -446,3 +446,11 @@ print("hi")
 --- err
 ERROR: options --rr and --valgrind cannot be specified at the same time.
 --- ret: 25
+
```

Comment:
Style: we need 3 blank lines between test blocks. BTW, we use the [reindex](https://github.com/openresty/openresty-devel-utils/blob/master/reindex) script to automatically format such `.t` test files for us.